MacOs/CMakeLists.txt

```diff
@@ -21,15 +16,18 @@ set(MAIN_FILES
         main.cpp
         )
 
-include_directories(Headers ../Shared/Interfaces ../Shared/Headers ../Library ../Shared/Proto/ProtoCompiled ${PROTOBUF_INCLUDE_DIR})
+include_directories(Headers ../Shared/Interfaces ../Shared/Headers ../Library ../Shared/Proto/ProtoCompiled)
 
 add_library(daemon_mac_lib STATIC ${DEAMON_MAC_SOURCE_FILES})
 
 add_executable(daemon_mac ${MAIN_FILES})
+set_target_properties(daemon_mac PROPERTIES OUTPUT_NAME "focus-daemon")
 
-target_link_libraries(daemon_mac daemon_mac_lib daemon_shared_lib ${PROTOBUF_LIBRARY} ${CARBON_LIBRARY})
+target_link_libraries(daemon_mac daemon_mac_lib daemon_shared_lib ${CARBON_LIBRARY})
 
-add_custom_command(TARGET daemon_mac POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_SOURCE_DIR}/MacOs/printAppTitle.scpt" "$<TARGET_FILE_DIR:daemon_mac>")
+add_custom_command(TARGET daemon_mac POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_SOURCE_DIR}/MacOs/printAppTitle.scpt" "$<TARGET_FILE_DIR:daemon_mac>/bin/printAppTitle.scpt")
```

Comment:
Doesn't work on CLION because it search in `$<TARGET_FILE_DIR:daemon_mac>` not `$<TARGET_FILE_DIR:daemon_mac>/bin`